astropy/cosmology/parameter.py

```diff
@@ -47,7 +47,7 @@ class Parameter:
     _registry_validators = {}
 
     def __init__(self, *, derived=False, unit=None, equivalencies=[],
-                 fvalidate="default", fmt=".3g", doc=None):
+                 fvalidate="default", fmt="g", doc=None):
```

Comment:
Does one need `fmt="g"`? Or is it just the default and one might as well do `fmt=""`?

Reply:
I think the latter. Changing now.

Reply:
So they aren't quite the same ``fmt=""`` doesn't turn ``0.0`` -> ``0``, like ``fmt="g"``. But I think that's an improvement in this case since it better reflects the actual dtype.